docs/book/src/advanced/traits.md

```diff
@@ -118,7 +118,7 @@ trait Trait {
 }
 ```
 
-Check the `associated consts` section on [constants](../basics/constants.md) page.
+Check the `associated constants` section on [constants](../basics/constants.md) page.
```

Comment:
Correct, but I think this should be changed to link directly to the section.

Could you please change this to be the following and link directly to that section:

    See `Associated Constants` for more information.

While you're here, could you also do the next section too:

  See `Associated Types` for more information.